proto/spacemesh/v1/smesher_types.proto

```diff
@@ -4,8 +4,14 @@ option go_package = "github.com/spacemeshos/api/release/go/spacemesh/v1";
 import "spacemesh/v1/types.proto";
 import "google/rpc/status.proto";
 
-message IsSmeshingResponse {
-    bool is_smeshing = 1;
+message SmeshingStatusResponse {
```

Comment:
I would change this method name to indicate its meaning - the post init setup status. The broader `smeshing` status is dependent on activations related flow. Please see #139 regarding providing data for the user via additional Activation data object data items. So, this method should be called something more concrete like `PosInitStatus()`.

Reply:
It’s currently covers both the post init setup status and the Smeshing boolean status.
We previously had `PostStatus()`. Are you suggesting to return it on a limited scope, in addition to having the Smeshing boolean status method?